lib/puppet/x/jenkins/provider/cli.rb

```diff
@@ -27,7 +27,6 @@ def self.inherited(subclass)
   initvars
 
   commands java: 'java'
-  confine feature: :retries
```

Comment:
why is that feature now gone?
does the below function not provide that feature?

Reply:
Previously the feature was provided by the gem. Now it uses functionality built into Puppet itself.

Reply:
you mean Ruby?
`retry` is a Ruby keyword (that i only learned about yesterday)

Reply:
Yes I did.